README-rsync.md

```diff
@@ -0,0 +1,235 @@
+
+```
+sudo rsync -aHq /Users/mac705/ /Users/macos-homebrew-ocaml-4.14
+```
+
+And use this from stage in your spec file `(from macos-homebrew-ocaml-4.14)`. Note that the OCluster PR still uses the Docker fetcher in the PR, but not on the machines. The machines have the following small patch that hopefully in the future can be removed.
+
+```diff
+diff --git a/obuilder b/obuilder
+--- a/obuilder
++++ b/obuilder
+@@ -1 +1 @@
+-Subproject commit 9c910f0693e938b491d3ba5ba09b43a4469b7c2d
++Subproject commit 9c910f0693e938b491d3ba5ba09b43a4469b7c2d-dirty
+diff --git a/worker/obuilder_build.ml b/worker/obuilder_build.ml
+index 9cebd2b..0a0af6d 100644
+--- a/worker/obuilder_build.ml
++++ b/worker/obuilder_build.ml
+@@ -22,7 +22,7 @@ type t = {
+ }
+
+ module Sandbox = Obuilder.Sandbox
+-module Fetcher = Obuilder.Docker
++module Fetcher = Obuilder.User_temp
+
+ let ( / ) = Filename.concat
+```
+
+The Docker `FETCHER` does work, but we don't have a [base-images](https://github.com/ocurrent/docker-base-images) equivalent to keep things up to date yet. To make a compatible Docker image, you can run `docker build` with the following Dockerfile:
+
+```Dockerfile
+FROM scratch
+COPY ./mac705-macos-homebrew-ocaml-4.14 /
+CMD [ "/bin/bash" ]
+```
+
+And push the image.
+
+## Ansible Playbook
+
+The Ansible playbook can be used to deploy Mac workers.  The following prerequisites should be satisfied:
+
+- Security & Privacy \ General \ Require Password -- disables screensaver
+- Users & Groups \ Login Options \ Automatic login as administrator
+- Sharing \ Screen sharing -- enables VNC
+- Sharing \ Remote login -- enables SSH.  Also select the “Allow full disk access for remote users” checkbox.
+- Energy Saver \ Prevents your Mac from automatically sleeping
+- Energy Saver \ Start up automatically after power failure
+- Install Apple Developer Command Line Tools
+- Turn off SIP by entering Recovery Mode (Intel: Command-R; M1: hold power button) `csrutil disable`
+- Install [macFUSE](https://osxfuse.github.io), which requires approval via System Preferences and a reboot of the system.
+  Use `brew install --cask macfuse` which will require a Homebrew install.
+- Install [Docker Desktop for Mac](https://docs.docker.com/desktop/mac/install/)
+- Set Docker to automatically start at sign in
+- Add the capability file for the scheduler pool you want to join to `./secrets/pool-macos-[arch].cap` (creating the `secrets` directory in the project root). For more info on the scheduler, see [ocurrent/ocluster](https://github.com/ocurrent/ocluster).
+- Add your SSH key to the `~/.ssh/authorized_keys` and update your `~/.ssh/config` so that you can SSH to the Mac without prompting for a username:
+
+```
+Host mac-mon-*
+	User administrator
+```
+
+Run the playbook as follows.  I have used `--limit` to target a single worker.
+
+```sh=
+ansible-playbook -i hosts --limit i7-worker-01 playbook.yml
+```
+
+## Starting and Stopping
+
+`ocluster-worker` is run via LaunchAgent.
```

Comment:
```suggestion
`ocluster-worker` is run via LaunchAgent.
```
Should this be OCluster-Worker or the command `ocluster-worker`. Seems like the latter. It was written Ocluster-worker, but since it's being run via LaunchAgent, I figured it was a comman: `ocluster-worker`.